doc/essentials/EnergyPlusEssentials.tex

```diff
@@ -0,0 +1,2260 @@
+``!-'' as an indicator. The input objects can be in any order. An
+example of an input object in an IDF file is shown below: 
+\begin{lyxcode}
+Building,
+
+~~Simple~One~Zone,~~~!-~Name
+
+~~0,~~~~~~~~~~~~~~~~~!-~North~Axis~\{deg\}
+
+~~Suburbs,~~~~~~~~~~~!-~Terrain
+
+~~0.04,~~~~~~~~~~~~~~!-~Loads~Convergence~Tolerance~Value
+
+~~0.004,~~~~~~~~~~~~~!-~Temperature~Convergence~Tolerance~Value~\{deltaC\}
+
+~~MinimalShadowing,~~!-~Solar~Distribution
+
+~~30,~~~~~~~~~~~~~~~~!-~Maximum~Number~of~Warmup~Days
+
+~~6;~~~~~~~~~~~~~~~~~!-~Minimum~Number~of~Warmup~Days
+\end{lyxcode}
+The details of this example input object are not important, but the
+use of commas, exclamation points, and the closing semi-colon are
+important. The IDF format is currently the most commonly used format
+throughout the EnergyPlus ecosystem of utilities and GUIs. The list
+of possible input objects and fields is documented in the Energy+.idd
+file.
+
+A variation on the IDF file format is the IMF file format which includes
+macros that can be used for parametric analysis or file management
+called EP-Macros. To learn more about macros see the Input Macros
+chapter of the AuxiliaryPrograms document.
+
+\subsubsection*{epJSON}
+
+A new file format based on the industry standard \href{https://www.json.org/}{JSON}
+format most often used to transmit data to and from web servers and
+web-browser based applications. It is a text-based file format. The
+JSON format has wide usage across many industries and is supported
+in just about every modern programming language. It is a field-value
+style format using brackets and colons to indicate the hierarchy and
+commas to separate each field and value pair. The input objects must
+appear grouped by the type of input object. The list of possible input
+objects and fields is documented in the Energy+.schema.epJSON file
+which uses \href{http://json-schema.org/}{json-schema}. The same
+input object shown above in IDF format is shown below in epJSON format:
+\begin{lyxcode}
+\{
+
+~~~~\textquotedbl Building\textquotedbl :~\{
+
+~~~~~~~~\textquotedbl Simple~One~Zone:~\{
+
+~~~~~~~~~~~~\textquotedbl idf\_max\_extensible\_fields\textquotedbl :~0,
+
+~~~~~~~~~~~~\textquotedbl idf\_max\_fields\textquotedbl :~8,
+
+~~~~~~~~~~~~\textquotedbl idf\_order\textquotedbl :~3,
+
+~~~~~~~~~~~~\textquotedbl loads\_convergence\_tolerance\_value\textquotedbl :~0.04,
+
+~~~~~~~~~~~~\textquotedbl maximum\_number\_of\_warmup\_days\textquotedbl :~30,
+
+~~~~~~~~~~~~\textquotedbl minimum\_number\_of\_warmup\_days\textquotedbl :~6,
+
+~~~~~~~~~~~~\textquotedbl north\_axis\textquotedbl :~0,
+
+~~~~~~~~~~~~\textquotedbl solar\_distribution\textquotedbl :~\textquotedbl MinimalShadowing\textquotedbl ,
+
+~~~~~~~~~~~~\textquotedbl temperature\_convergence\_tolerance\_value\textquotedbl :~0.004,
+
+~~~~~~~~~~~~\textquotedbl terrain\textquotedbl :~\textquotedbl Suburbs\textquotedbl{}
+
+~~~~~~~~\}
+
+~~~~\}
+
+\}
+\end{lyxcode}
```

Comment:
I'm guessing these lyxcode blocks will have to be cleaned up into regular code/verbatim blocks as in the other E+ docs, once we stop using LyX.